src/cursor/cur_hs.c

```diff
@@ -528,6 +528,21 @@ __curhs_next_visible(WT_SESSION_IMPL *session, WT_CURSOR_HS *hs_cursor)
     return (ret);
 }
 
+/*
+ * __wt_curhs_time_window --
+ *     Get the underlying time window of the hs cursor.
+ */
+void
+__wt_curhs_time_window(WT_CURSOR *hs_cursor, WT_TIME_WINDOW **twp)
```

Comment:
I remember seeing similiar function __wt_hs_upd_time_window  in other PR, request to check if this is redundant ?

Reply:
This is redundant and I just copy pasted to make it work. This will go away once I'll merge the changes from the other PR.